source/tests/pt/model/water/multitask_sharefit.json

```diff
@@ -0,0 +1,133 @@
+{
+  "model": {
+    "shared_dict": {
+      "my_type_map": [
+        "O",
+        "H",
+        "B"
+      ],
+      "my_descriptor": {
+        "type": "se_e2_a",
+        "sel": [
+          46,
+          92
+        ],
```

Comment:
Why is the size of `my_type_map` 3 and the size of `sel` 2?

Reply:
Oh I copied from `pt/model/water/multitask.json` and it's a typo. And I found that the descriptor part will be replaced by other templates, so the settings here has no effect. https://github.com/iProzd/deepmd-kit/blob/share_fit_prod/source/tests/pt/test_multitask.py#L203-L205
Fixed.